packages/network/src/logic/tracker/Tracker.ts

```diff
@@ -232,15 +232,29 @@ export class Tracker extends EventEmitter {
         }
         if (this.overlayPerStreamPart[streamPartId]) {
             const instructions = this.overlayPerStreamPart[streamPartId].formInstructions(node, forceGenerate)
-            Object.entries(instructions).forEach(async ([nodeId, newNeighbors]) => {
-                const counterValue = this.instructionCounter.setOrIncrement(nodeId, streamPartId)
-                await this.instructionSender.addInstruction({
-                    nodeId,
+
+            // Send empty instruction if and only if the node is alone in the topology
+            if (this.overlayPerStreamPart[streamPartId].hasNode(node)
+                && this.overlayPerStreamPart[streamPartId].getNumberOfNodes() === 1
+                && Object.keys(instructions).length === 0) {
+                this.instructionSender.bypassBuffer({
```

Comment:
Why do we need to bypass the buffer? Could use an explanation, as it is not immediately obvious.

Reply:
There's a chance that this is not required anymore I'll test it out.

Reply:
Removed the bypass functionality as it was not necessary